test/spot-checker.js

```diff
@@ -49,12 +65,29 @@ test('fetchCAR - graphsync', async () => {
     address: addr,
     cid,
     stats,
+    dagScope: 'block'
+  })
+
+  assertEquals(stats.blockRetrieval.statusCode, 200, 'stats.statusCode')
+  assertEquals(stats.blockRetrieval.timeout, false, 'stats.timeout')
+  assertEquals(stats.blockRetrieval.byteLength, 120, 'stats.byteLength')
```

Comment:
Ditto.

Reply:
Addressed in https://github.com/CheckerNetwork/spark-spot-check/pull/6/commits/b5068e3e28f1e841701b98921cf48ef9d2a46b5a